src/utils/fs.rs

```diff
@@ -95,13 +95,15 @@ pub fn copy_files_except_ext(
     from: &Path,
     to: &Path,
     recursive: bool,
+    avoid_dir: Option<&PathBuf>,
```

Comment:
This is a public API, so this is a breaking change.  However, I don't see anyone externally using it.  

@Dylan-DPC, how do you feel about being stringent about API compatibility like this?

Reply:
Let me know what you want to do here - don't see an easy way to make this PR work without this or a similar change.... so when OK-ed by you guys I'll rectify these three comments.

Reply:
@ehuss I should have some time this week and would like to get back to this PR and close it if possible. Can you let me know what you want to do about the API and breaking change in order to resolve it? Thanks

Reply:
Hm, I think it would be ok to just change it. We can change it back if it causes problems for anyone.